.github/workflows/run-unit-tests.yml

```
@@ -140,6 +151,7 @@ jobs:
       PYDANTIC: "${{ inputs.pydantic }}"
       PYTHON_MAJOR_MINOR_VERSION: "${{ matrix.python-version }}"
       UPGRADE_BOTO: "${{ inputs.upgrade-boto }}"
+      AIRFLOW_MONITOR_DELAY_TIME_IN_SECONDS: "${{inputs.monitor-delay-time-in-seconds}}"
```

Jinja is not my forte, so maybe it doesn't matter or maybe there is a reason it is different here, but the other lines around this have spaces between the curly braces and the text inside them.

```suggestion
      AIRFLOW_MONITOR_DELAY_TIME_IN_SECONDS: "${{ inputs.monitor-delay-time-in-seconds }}"
```

---

It's not JINJA - it's GitHub Actions expression language :). It does not matter really, if spaces are there - would be good for consistency, but I think we already use it differently in different places .. as long as we won't be able to have some linting to add it, it does not bother me too much :)